gammapy/data/gti.py

```diff
@@ -190,27 +258,29 @@ def time_intervals(self):
         ]
 
     @classmethod
-    def from_time_intervals(cls, time_intervals, reference_time="2000-01-01"):
+    def from_time_intervals(cls, time_intervals, reference_time=None):  # "2000-01-01"):
         """From list of time intervals
 
         Parameters
         ----------
         time_intervals : list of `~astropy.time.Time` objects
             Time intervals
         reference_time : `~astropy.time.Time`
-            Reference time to use in GTI definition
+            Reference time to use in GTI definition. Default is None.
+            If None, use minimum start time.
 
         Returns
         -------
         gti : `GTI`
             GTI table.
         """
-        reference_time = Time(reference_time)
-        start = Time([_[0] for _ in time_intervals]) - reference_time
-        stop = Time([_[1] for _ in time_intervals]) - reference_time
-        meta = time_ref_to_dict(reference_time)
-        table = Table({"START": start.to("s"), "STOP": stop.to("s")}, meta=meta)
-        return cls(table=table)
+        start = Time([_[0] for _ in time_intervals])
+        stop = Time([_[1] for _ in time_intervals])
+
+        if reference_time is None:
+            reference_time = start.min()
```

Comment:
The behavior is well documented, but maybe we should still use the same `TIME_REF_DEFAULT` instead?